tapir/coop/emails/co_purchaser_updated_mail.py

```diff
@@ -0,0 +1,64 @@
+from typing import List
+
+from django.utils.translation import gettext_lazy as _
+
+from tapir.accounts.models import TapirUser
+from tapir.coop.models import ShareOwner
+from tapir.core.tapir_email_base import TapirEmailBase
+
+
+class CoPurchaserUpdatedMail(TapirEmailBase):
+    def __init__(self, tapir_user: TapirUser):
+        super().__init__()
+        self.tapir_user = tapir_user
+
+    @classmethod
+    def get_unique_id(cls) -> str:
+        return "tapir.coop.co_purchaser_updated_mail"
+
+    @classmethod
+    def get_name(cls) -> str:
+        return _("Co-purchaser updated")
+
+    @classmethod
+    def get_description(cls) -> str:
+        return _(
+            "Sent to a member when their new co-purchaser gets registered on their profile."
+        )
+
+    def get_subject_templates(self) -> List:
+        return [
+            "coop/email/co_purchaser_updated.subject.html",
+            "coop/email/co_purchaser_updated.subject.default.html",
+        ]
+
+    def get_body_templates(self) -> List:
+        return [
+            "coop/email/co_purchaser_updated.body.html",
+            "coop/email/co_purchaser_updated.body.default.html",
+        ]
+
+    @classmethod
+    def get_dummy_version(cls) -> TapirEmailBase | None:
```

Comment:
What's the meaning of 'dummy' in this context? I see it was already used in different places, but I couldn't get it.